templates/default/invitations.js

```diff
@@ -3,166 +3,191 @@
-				} else if (!self.filtering) {
-						$('#xoct_filter').remove();
-				}
-		});
-
-
-		$(document).on('click', '#xoct_filter', function () {
-				self.filter_container.val('');
-				self.filter('');
-				$(this).remove();
-		});
-
-		$(document).on('click', '#xoct_invite_all', function() {
-			self.inviteAll();
-		})
-
-	},
-	clear: function () {
-		this.container_invited.empty();
-		this.container_available.empty();
-	},
-
-	load: function () {
-		var self = this;
-		this.before_load();
-		var url = this.data_url;
-		$.ajax({url: url, type: "GET", data: {"cmd": "getAll"}}).done(function (data) {
-			self.clear();
-			for (var i in data.available) {
-				self.container_available.append('<li class="list-group-item xoct_participant_available" data-invitation-id="' + data.available[i].id + '">'
-					+ '<div style="margin-right:30px;">'
-					+ data.available[i].name
-					+ '</div>'
-					+ '<button class="btn btn-primary xoct_invite pull-right"><span class="glyphicon glyphicon-plus"></span></button>'
-					+ '</li>');
-			}
-
-			for (var i in data.invited) {
-				self.container_invited.append('<li class="list-group-item" data-invitation-id="' + data.invited[i].id + '">'
-					+ '<div style="margin-right:30px;">'
-					+ data.invited[i].name
-					+ '</div>'
-					+ '<button class="btn btn-default xoct_remove pull-right"><span class="glyphicon glyphicon-minus"></span></button>'
-					+ '</li>');
-			}
-
-			if (!data.available || data.available.length == 0) {
-				self.container_available.html('<li class="list-group-item">' + self.lng.none_available + '</li>');
-			}
-
-			if (!data.invited || data.invited.length == 0) {
-				self.container_invited.html('<li class="list-group-item">' + self.lng.none_available + '</li>');
-			}
-
-			self.after_load();
-		});
-	},
-
-	addInvitation:function (id) {
-		var url = this.data_url;
-		var self = this;
-		this.before_load();
-
-		$.ajax({url: url + "&cmd=create", type: "POST", data: {"id": id}}).done(function (data) {
-			self.load();
-			self.after_load();
-		});
-	},
-
-	inviteAll: function() {
-		var ids = [];
-		$('#xoct_available li:visible').each(function(i) {
-			ids.push($(this).attr('data-invitation-id'));
-		});
-
-		var url = this.data_url;
-		var self = this;
-		this.before_load();
-
-		$.ajax({url: url + "&cmd=createMultiple", type: "POST", data: {"ids": ids}}).done(function (data) {
-			self.load();
-			self.after_load();
-		});
-	},
-
-	removeInvitation:function (id) {
-		var url = this.data_url;
-		var self = this;
-		this.before_load();
-		$.ajax({url: url + "&cmd=delete", type: "POST", data: {"id": id}}).done(function (data) {
-			$('[data-invitation-id="' + id + '"]').remove();
-			self.load();
-			self.after_load();
-		});
-	},
-
-    /**
+const xoctInvitation = {
+  selected_id: 0,
+  data_url: '',
+  container: null,
+  filter_container: null,
+  filtering: false,
+  lng: [
```

Comment:
I understand that it's written as array in all the other files as well, but it should actually be `object`, since `this.lng` is consistently accessed using dot notation throughout the code.
This could lead to issues if `lngFromJson` hasn't been called beforehand.
NOTE: This also applies to other places!